prime-router/docs/proposals/0019-server-server-authentication-updates.md

```diff
@@ -0,0 +1,36 @@
+## Background
```

Comment:
Instead of having a document for these specific updates, I'm wondering if we need a design doc that the content here is a part of? Maybe this document would include the `how-to-use-token-auth.md` document?

My thought is that we would have a .MD document for each "feature", with all relevant info for that feature in the document, so things are not spread out. If you check the latest master, there should be a prime-router/docs/features folder, I recommend we add to that?

Reply:
If you want to do this work as a follow-on ticket, please create the ticket under https://github.com/CDCgov/prime-reportstream/issues/8913 and link here, please.